lib/resources/example_json/r4_examples_account.rb

```diff
@@ -0,0 +1,39 @@
+module Cerner
+  module Resources
+
+    R4_ACCOUNT ||= {
+      "resourceType": "Account",
+      "id": "1527935",
+      "text": {
+          "status": "generated",
+          "div": "<div><p><b>Account</b></p><p><b>Service Period Start Date</b>: Jun 29, 2009  1:27 P.M. CDT</p><p><b>Service Period End Date</b>: May 14, 2014  7:00 P.M. CDT</p><p><b>Account Type</b>: guarantor-balance</p></div>"
```

Comment:
The div needs to be HTML escaped

Reply:
https://github.com/cerner/fhir.cerner.com/commit/62764511c7f719faa5141a6b7bcbfcc839226e9f#diff-139e4f82fa560c1b721301666faf1a3eR9

The read one is HTML escaped: +1